python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -912,3 +935,21 @@ def _summarize(self, exposure, stars, background):
         # applied calibration). This needs to be checked.
         summary = self.compute_summary_stats.run(exposure, stars, background)
         exposure.info.setSummaryStats(summary)
+
+    def _recordFlaggedPixelFractions(self, exposure):
```

Comment:
For consistency I'd always call pixels Masked rather than Flagged. (though admittedly "mask" is a bad holdover from the era of only having a single "mask" plane, and that keeps confusing new users)